manager/apis/app/v1alpha1/blueprint_types.go

```diff
@@ -103,9 +103,9 @@ type ComponentTemplate struct {
 	// +required
 	Kind string `json:"kind"`
 
-	// Resources contains the location of the helm chart with info detailing how to deploy
+	// Resource contains the location of the helm chart with info detailing how to deploy
 	// +required
-	Resources []string `json:"resources"`
+	Resource string `json:"resource"`
```

Comment:
Shouldn't we name this chart instead of resource?

Reply:
will change